src/cli/stories.py

```
@@ -0,0 +1,190 @@
+            return lines[idx + 1].strip()
+    raise StoryValidationError(f"Unable to determine status for {story_path}")
+
+
+def _story_identifier(story_path: Path) -> str:
+    return story_path.stem.split(".")[0:2][0] if "." not in story_path.stem else ".".join(story_path.stem.split(".")[0:2])
+
+
+def _load_latest_story(stories_dir: Path, *, exclude: Optional[Path] = None) -> StoryInfo:
+    stories = _discover_stories(stories_dir, exclude=exclude)
+    if not stories:
+        raise StoryValidationError("No existing stories found.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)